app/metrics.yaml

```diff
@@ -5022,6 +5023,141 @@ top_sites:
     notification_emails:
       - android-probes@mozilla.com
     expires: 111
+  context_id:
+    type: uuid
+    description: |
+      A UUID that is unjoinable with other browser metrics. This ID will not be
+      shared with AdM, only for internal uses. This ID is shared across all
+      contextual services features.
+    lifetime: user
```

Comment:
Have we checked with Glean team that we should use `lifetime: user` here?  Can we avoid doing that since there's no possible way of removing this value later (unless user uninstall/clear storage)

Reply:
Yes, I was the one who suggested this. I don't think there will be a need to remove this id, and hopefully this can be addressed inside of Glean to properly remove expired user lifetime metrics (there's a bug for it).

Reply:
I am planning to look at this again and just call `set()` with the stored `contileContextId`

Reply:
Then, ping lifetime would be appropriate if the application will manage the stored id